Resources/Prototypes/SimpleStation14/Objectives/freewillObjectives.yml

```diff
@@ -0,0 +1,106 @@
+- type: objective
+  id: FreewillAddictObjective
+  issuer: Your free will
+  difficultyOverride: 2.75
+  conditions:
+    - !type:GenericFreewillCondition
+      title: objective-condition-addict-title
+      description: objective-condition-addict-description
+      icon:
+        sprite: Clothing/Hands/Gloves/fingerless.rsi
+        state: icon
+
+- type: objective
+  id: FreewillDealerObjective
+  issuer: Your free will
+  difficultyOverride: 2.75
+  conditions:
+    - !type:GenericFreewillCondition
+      title: objective-condition-dealer-title
+      description: objective-condition-dealer-description
+      icon:
+        sprite: Clothing/Hands/Gloves/fingerless.rsi
+        state: icon
+
+- type: objective
+  id: FreewillGraffitiArtistObjective
+  issuer: Your free will
+  difficultyOverride: 2.75
+  conditions:
+    - !type:GenericFreewillCondition
+      title: objective-condition-graffiti-artist-title
+      description: objective-condition-graffiti-artist-description
+      icon:
+        sprite: Clothing/Hands/Gloves/fingerless.rsi
+        state: icon
+
+- type: objective
+  id: FreewillLitterbugObjective
+  issuer: Your free will
+  difficultyOverride: 2.75
+  conditions:
+    - !type:GenericFreewillCondition
+      title: objective-condition-litterbug-title
+      description: objective-condition-litterbug-description
+      icon:
+        sprite: Clothing/Hands/Gloves/fingerless.rsi
+        state: icon
+
+- type: objective
+  id: FreewillObsessedObjective
+  issuer: Your free will
+  difficultyOverride: 2.75
+  conditions:
+    - !type:GenericFreewillCondition
+      title: objective-condition-obsessed-title
+      description: objective-condition-obsessed-description
+      icon:
+        sprite: Clothing/Hands/Gloves/fingerless.rsi
```

Comment:
icon, a heart?

Reply:
Ooo, that would've been good. I went with a blindfold...
Bit darker, I suppose :P